Author: aszlig <aszlig@nix.build>
Date:   Wed Mar 28 07:23:59 2018 +0200

    server: Don't rely on package.json for version
    
    We're going to inject the right Habitica version using Nix, so we don't
    need another import.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: hardcoded-server-version.patch

diff --git a/website/server/middlewares/response.js b/website/server/middlewares/response.js
index 420ba9e529..1769d57ded 100644
--- a/website/server/middlewares/response.js
+++ b/website/server/middlewares/response.js
@@ -1,4 +1,3 @@
-import packageInfo from '../../../package.json';
 import {
   model as UserNotification,
 } from '../models/userNotification';
@@ -20,7 +19,7 @@ module.exports = function responseHandler (req, res, next) {
       response.userV = user._v;
     }
 
-    response.appVersion = packageInfo.version;
+    response.appVersion = '@HABITICA_VERSION@';
 
     res.status(status).json(response);
   };
